src/game-support/yakuza-6.md

```diff
@@ -0,0 +1,13 @@
+# Yakuza 6: The Song of Life
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Silver installs=Yes opens=Yes}}
```

Comment:
Glad to see a new contributor :) Are there any glitches/errors during a game run? If no then it should be a Gold status as the configuration does not require much technical skills.

Reply:
He said hair doesn’t work.

Reply:
Please add this info under the "warning" section.

Reply:
@mattmaniak it's not required.

Reply:
I know but it is quite helpful to know what exactly may be broken in a game that someone wants to play.

Reply:
i've asked them in the discord, if they reply soon i'll just directly add what he said. otherwise, i'll paraphrase and test myself when i get time. for right now, while referring to a screenshot, he said "Look at the hair. It's gone. On top of that, special effects from the game are also missing" while using DXVK.

i just want to confirm whether or not that still happens while d3dmetal is being used and get more of an "official" word from him

Reply:
Actually, that’s for Kiwami 2. One way to fix this is to disable it and set the Enhanced Sync option to MSync.

Reply:
> Actually, that’s for Kiwami 2. One way to fix this is to disable it and set the Enhanced Sync option to MSync.

whats for kiwami 2? and are you saying that the hair and effects stuff is broken when DXVK is enabled and MSync is turned on?

Reply:
Yeah.

Reply:
I see that people are confusing Yakuza 6 for Kiwami 2 for some reason. 
![Uploading Screenshot 2024-10-12 at 23.24.46.png…]()

Reply:
This picture is from Kiwami 2.

Reply:
DXVK on and Esync was the bottle config.